org.eclipse.jdt.debug.tests/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.debug.tests; singleton:=true
-Bundle-Version: 3.12.550.qualifier
+Bundle-Version: 3.12.650.qualifier
```

Comment:
This looks wrong. I don't know how 550 ended up in master (IMO it's failure from merging BETA_JAVA* branch FYI @jarthana ) . The bundle version should be 3.12.600 here.